apps/condo/domains/marketplace/hooks/useFilters.tsx

```diff
@@ -0,0 +1,62 @@
+import {
+    MeterReadingWhereInput,
+    MeterReadingSource as MeterReadingSourceType,
+    MeterResource as MeterResourceType,
+    PaymentWhereInput,
+} from '@app/condo/schema'
+import compact from 'lodash/compact'
+import get from 'lodash/get'
+import { useMemo } from 'react'
+
+import { useIntl } from '@open-condo/next/intl'
+import { useOrganization } from '@open-condo/next/organization'
+
+import {
+    FiltersMeta,
+} from '@condo/domains/common/utils/filters.utils'
+import {
+    getDayRangeFilter,
+    getStringContainsFilter,
+} from '@condo/domains/common/utils/tables.utils'
+import { MARKETPLACE_PAGE_TYPES, MarketplacePageTypes } from '@condo/domains/marketplace/utils/clientSchema'
+
+
+const addressStringContainsFilter = getStringContainsFilter(['invoice', 'property', 'address'])
+const invoiceNumberFilter = getStringContainsFilter(['invoice', 'number'])
+const ticketNumberFilter = getStringContainsFilter(['invoice', 'ticket', 'number'])
+const statusFilter = getStringContainsFilter('status')
+const paymentDateRangeFilter = getDayRangeFilter('date')
+
+export function useFilters (marketplacePageType: MarketplacePageTypes): Array<FiltersMeta<PaymentWhereInput>>  {
```

Comment:
We need 3 different useFilters/useTableColumns for 3 tabs (invoices/payments/marketItems).
Use it for payments only pls (remove switch/case)